interrupt-test.c

```diff
@@ -0,0 +1,90 @@
+#include <stdlib.h>
+#include "quickjs.h"
+
+#define MAX_TIME 10
+
+#define expect(condition) \
+    do { \
+        if (!(condition)) { \
+            fprintf(stderr, "Failed: %s, file %s, line %d\n", \
+                    #condition, __FILE__, __LINE__); \
+            exit(EXIT_FAILURE); \
+        } \
+    } while (0)
+
+static int timeout_interrupt_handler(JSRuntime *rt, void *opaque) 
+{
+    int *time = (int *)opaque;
+    if (*time <= MAX_TIME) {
+        *time += 1;
+    }
+    return *time > MAX_TIME;
+}
+
+static void sync_call()
```

Comment:
```suggestion
static void sync_call(void)
```
In C before C23, `()` means a function taking zero or more arguments.

In C23, `()` means zero arguments - but quickjs is a C11 project so that doesn't apply.